simpleviewinc/tap-events-force
# WIP? -- ZEN-495: No Sessions Message

**Ticket**: [ZEN-495](https://jira.simpleviewtools.com/browse/zen-495)

> I'm still waiting on Ben to respond to my questions about the exact message and styling of this component.

## Context

* When there are no sessions to display for a day, we should show a message explaining that

## Goal

* Implement the component and it's styling

## Updates

* `src/components/grid/emptyDayMessage.js`
  * the component for the message
* `src/components/sessions.js`
  * updated the `AgendaSessions` component to use `EmptyDayMessage`
* `src/theme/components/grid/emptyDayMessage.js`
  * styles for the component

## Testing

* `keg evf pack run package=docker.pkg.github.com/simpleviewinc/keg-packages/tap:zen-495-no-sessions-msg`
* navigate to http://evf-zen-495-no-sessions-msg.local.kegdev.xyz/?state=emptyDay
* move to day 2 
* verify the message appears
  * repeat that for mobile web
* move to day 1
* open the filter modal and filter on "Waiting List"
* verify the same message appears
